internal/cli/cmd/sysdump.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2021 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"k8s.io/klog/v2"
+
+	"github.com/cilium/cilium-cli/sysdump"
+)
+
+var (
+	d bool
+	o = sysdump.SysdumpOptions{
+		LargeSysdumpAbortTimeout: sysdump.DefaultLargeSysdumpAbortTimeout,
+		LargeSysdumpThreshold:    sysdump.DefaultLargeSysdumpThreshold,
+	}
+	t time.Duration
```

Comment:
`d`, `o` and `t` are one letter vars that are global to the `cmd` package. It would be good to avoid this by, for example, creating one global struct for the sysdump command options.